pkg/controller/oneagent/oneagent_controller.go

```diff
@@ -363,18 +364,24 @@ func newPodSpecForCR(instance *dynatracev1alpha1.OneAgent) corev1.PodSpec {
 // Returns an error in the following conditions:
 //  - failure on object deletion
 //  - timeout on waiting for ready state
-func (r *ReconcileOneAgent) deletePods(instance *dynatracev1alpha1.OneAgent, pods []corev1.Pod) error {
+func (r *ReconcileOneAgent) deletePods(reqLogger logr.Logger, instance *dynatracev1alpha1.OneAgent, pods []corev1.Pod) error {
 	for _, pod := range pods {
+		reqLogger.Info("deleting pod", "pod", pod.Name)
+
 		// delete pod
 		err := r.client.Delete(context.TODO(), &pod)
 		if err != nil {
 			return err
 		}
 
+		reqLogger.Info("waiting until pod is ready", "pod", pod.Name)
+
 		// wait for pod on node to get "Running" again
 		if err := r.waitPodReadyState(instance, pod); err != nil {
```

Comment:
I know the `err` is print after this function is called, but at least indicating, that an error while being in the delete loop happened, could help.
What do you think?

Reply:
If we're going to try to update all the nodes rather than exiting quickly, then yes, this is needed to show errors per pod.